File: ayush.dev/abhisheks+shared/GPDScGit_CloneExternal/pipelines/ScalaInsideProjectDatasetsOnly/code/src/main/scala/io/prophecy/pipelines/scalainsideprojectdatasetsonly/graph/src_delta.scala
package io.prophecy.pipelines.scalainsideprojectdatasetsonly.graph

import io.prophecy.libs._
import io.prophecy.pipelines.scalainsideprojectdatasetsonly.config.ConfigStore._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object src_delta {

  def apply(spark: SparkSession): DataFrame =
    spark.read
      .format("delta")
      .load("dbfs:/Prophecy/qa_data/delta/all_type_and_partition")

}

File: test/ayush.dev/py_prechange/pipelines/PyPipSharedDataset/code/pypipshareddataset/graph/src_delta.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from pypipshareddataset.config.ConfigStore import *
from pypipshareddataset.udfs.UDFs import *

def src_delta(spark: SparkSession) -> DataFrame:
    return spark.read.format("delta").load("dbfs:/Prophecy/qa_data/delta/all_type_and_partition")
